bundles/org.openhab.binding.sonnen/src/main/java/org/openhab/binding/sonnen/internal/SonnenBindingConstants.java

```diff
@@ -45,4 +45,10 @@ public class SonnenBindingConstants {
     public static final String CHANNELFLOWGRIDBATTERYSTATE = "flowGridBatteryState";
     public static final String CHANNELFLOWPRODUCTIONBATTERYSTATE = "flowProductionBatteryState";
     public static final String CHANNELFLOWPRODUCTIONGRIDSTATE = "flowProductionGridState";
+
+    // List of new Channel ids for PowerMeter API
+    public static final String ENERGYIMPORTEDSTATEPRODUCTION = "energyImportedStateProduction";
+    public static final String ENERGYEXPORTEDSTATEPRODUCTION = "energyExportedStateProduction";
+    public static final String ENERGYIMPORTEDSTATECONSUMPTION = "energyImportedStateConsumption";
+    public static final String ENERGYEXPORTEDSTATECONSUMPTION = "energyExportedStateConsumption";
```

Comment:
```suggestion
    public static final String CHANNEL_ENERGY_IMPORTED_STATEP_RODUCTION = "energyImportedStateProduction";
    public static final String CHANNEL_ENERGY_EXPORTED_STATEP_RODUCTION = "energyExportedStateProduction";
    public static final String CHANNEL_ENERGY_IMPORTED_STATE_CONSUMPTION = "energyImportedStateConsumption";
    public static final String CHANNEL_ENERGY_EXPORTED_STATE_CONSUMPTION = "energyExportedStateConsumption";
```

Reply:
I add channel but i keep the naming equal to the other existing channels